collection.php

```diff
@@ -0,0 +1,39 @@
+<?php require('database.php'); ?>
+<html>
+<head>
+    <title>Collection</title>
+    <link rel="stylesheet" type="text/css" href="collection.css">
+</head>
+<body>
+<header>
+    <nav>
```

Comment:
This isn't actually a nav, I'd suggest removing the nav tag and just having the header.